lib/modules/dosomething/dosomething_global/dosomething_global.module

```diff
@@ -87,7 +87,17 @@ function dosomething_global_menu_alter(&$items) {
 }
 
 /**
- * Implements hook_meny_local_tasks_alter().
+ * Implements hook_form_node_admin_content_alter().
+ */
+function dosomething_global_form_node_admin_content_alter(&$form, &$form_state, $form_id) {
```

Comment:
since we don't care about this being for #global want to just put this in `dosomething_helpers`?